readme.md

```diff
@@ -163,12 +163,14 @@ Please install CUDA (v12.0|v11.4)(Visual Studio 2022|2019) from [official site](
     ```
     More details here: [vcpkg](https://github.com/microsoft/vcpkg).
 
-2. Execute install.bat
+2. (Optional, but recommended) Install [AWS CLI v2](https://awscli.amazonaws.com/AWSCLIV2.msi). Once installed, reopen PowerShell or CMD. This will allow you to use the vcpkg binary cache from our aws s3, which will speed up the installation process and reduce the required disk space.
```

Comment:
does it have cache for vs2019 *and* for vs2022 or only one of them, can it detect required version?